source/faq.txt

```diff
@@ -107,3 +107,20 @@ have more than 2 non-arbiter nodes and you must sync from a non-arbiter
 node. Use the source cluster's connection string to specify a
 :ref:`read preference <mongodb-uri>` for a non-arbiter, data-bearing
 node. 
+
+What If I see a Slow Operation Warning?
+---------------------------------------
+
+Slow operation warnings can occur during the initial sync or the application 
```

Comment:
[question]

Is there a threshold for "slow operations" that we can specify here, and can the user configure the threshold?

Reply:
@jeff-allen-mongo, I checked with Jian Guan.  The threshold is hard coded at 1 minute and subject to change, so best not to specify it here.